fastapi-sample2/main.py

```diff
@@ -0,0 +1,107 @@
+from fastapi import FastAPI,Depends,HTTPException
+
+from database_config import SessionLocal
+from crud_departments import (get_department as g_dept, get_departments as g_depts, insert_department as i_dept)
+from crud_users import ( get_users as g_users, get_user as g_user, insert_user as i_users)
+from sqlalchemy.orm import Session
+import validation
+from typing import List
+
+app = FastAPI()
+
+# DB接続
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+#departments
+
+# get
+@app.get("/departments",response_model=List[validation.Departments])
+async def get_departments(db:Session=Depends(get_db)):
```

Comment:
```suggestion
async def index_departments(db:Session=Depends(get_db)):
```
命名規則に悩まれたみたいですね！確かに`as`使うのも手段として良いと思います！
こちらの25行目はFastAPIが読み込む関数なため、実質我々エンジニアが呼び出すことはないと思われます。
ですのでこちらの関数名を変えてあげるのも一つ手段として良いかもしれません。
SGではRailsの命名規則を使って開発をしているのでよければ参考にしてみてください

[Rails on API](https://qiita.com/k-penguin-sato/items/adba7a1a1ecc3582a9c9)

Reply:
お送りした記事の`posts.controller.rb`のファイルで使用している命名規則が参考になりそうです🙇

Reply:
なるほどです。確かに使うことのない関数が一番シンプルな名前を奪っているのは良くないですね。
参考記事ありがとうございます。
記事を参考にしてもう一度リクエストいたします🙇